common/src/main/java/org/jboss/pnc/bacon/common/cli/AbstractGetSpecificCommand.java

```diff
@@ -36,10 +38,15 @@
     @Argument(required = true, description = "Id of item")
     private String id;
 
+    @Option(shortName = 'o', overrideRequired = false, hasValue = false, description = "use json for output (default to yaml)")
+    private boolean jsonOutput = false;
+
     @Override
     public CommandResult execute(CommandInvocation commandInvocation) throws CommandException, InterruptedException {
 
-        return super.executeHelper(commandInvocation, () -> System.out.println(getSpecific(id)));
+        return super.executeHelper(commandInvocation, () -> {
+            System.out.println(ObjectHelper.getOutputMapper(jsonOutput).writeValueAsString(getSpecific(id)));
```

Comment:
I think we could shorten the method call to:

```
ObjectHelper.print(<is json output?>, object);
```

It's shorter and it hides the implementation details of the formatter engine we are using (Jackson).

Reply:
Let me know what you think! :)

Reply:
:+1: